build-system/tasks/update-packages.js

```diff
@@ -98,6 +106,27 @@ function patchRegisterElement() {
   writeIfUpdated(patchedName, file);
 }
 
+/**
+ * Makes sure ES6 packages in node_modules that are used by the runtime will be
+ * transformed by babelify. This is a no-op if transforms are already enabled.
+ */
+function transformEs6Packages() {
+  es6Packages.forEach(es6Package => {
+    const packageJsonFile = 'node_modules/' + es6Package + '/package.json';
```

Comment:
@rsimha could you explain why this is better than the `global: true` option @choumx applied?

Reply:
@erwinmombay @choumx Let me know if you agree that transforming only those `node_modules` dependencies that we use is preferable to doing a global transform.